streams/src/test/java/org/apache/kafka/streams/tests/SmokeTestDriver.java

```diff
@@ -569,7 +569,7 @@ private static boolean verifyTAgg(final PrintStream resultStream,
                 }
 
                 if (entry.getValue().getLast().value().longValue() != expectedCount) {
-                    resultStream.println("fail: key=" + key + " tagg=" + entry.getValue() + " expected=" + expected.get(key));
```

Comment:
We removed `key` from the map above and store contents in `expectedCount`, should use that in message